web/src/providers/Span/Span.provider.tsx

```diff
@@ -47,7 +47,7 @@ interface IProps {
   children: React.ReactNode;
 }
 
-export const useSpan = () => useContext(Context);
+export const useSpan = (): IContext => useContext(Context);
```

Comment:
I think the returned type is already deducted by the editor, no need to have it be explicit

Reply:
i dont think so

Reply:
we could all do useContext<IContext>(Context) but it boils down to the same thing